BUILD.gn

```diff
@@ -476,6 +497,13 @@ action("generate_upstream_blink_version") {
   ]
 }
 
+if (is_linux) {
+  pkg_config("libnotify_config") {
```

Comment:
I think it is much cleaner to move the detection of libnotify to a separate file, just like Chromium itself does with `src/build/config/linux/{atk,dbus,gconf,...}`. This way, the only changes you need to make here is the `configs +=` part, and all the rest is done in another file with much fewer `if (is_linux)` checks.